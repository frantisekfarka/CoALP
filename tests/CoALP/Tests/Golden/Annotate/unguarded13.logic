q(G1,s(G2),X1,X2):-p(X1,X2,X1,X2).
p(t(G1),G2,X1,X2):-q(X1,X2,X1,X2).