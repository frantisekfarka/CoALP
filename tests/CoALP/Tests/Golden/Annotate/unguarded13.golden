q(V_2i, s(V_3i), V_4i, V_5i, transform-func-1(V_10c)) :- p(V_4i, V_5i, V_4i, V_5i, V_10c).
p(t(V_6i), V_7i, V_8i, V_9i, transform-func-2(V_11c)) :- q(V_8i, V_9i, V_8i, V_9i, V_11c).
